Jenkinsfile

```diff
@@ -25,8 +25,8 @@ export GIT_BRANCH=${env.GIT_BRANCH}
 export GIT_COMMIT=${env.GIT_COMMIT}
 export AZURE_TOKEN='${env.AZURE_TOKEN}'
 export MAPBOX_TOKEN='${env.MAPBOX_TOKEN}'
-# only use 1 thread for tici tests since most require HIL
-export PYTEST_ADDOPTS="-n 0"
+# only use 1 thread for tici tests since most require HIL, and print stdout for debugging
+export PYTEST_ADDOPTS="-n 0 -s"
```

Comment:
why isn't it showing once it fails?

Reply:
xdist doesn't let you capture stdout, it's very strange. only stderr is actually captured.

Reply:
https://github.com/pytest-dev/pytest-xdist/issues/354

Reply:
ah, it's the xdist issue. do we use xdist at all on the tici tests? we can just add `-n0`

Reply:
it's there

Reply:
`-n0 -s` seems to be enough to show stdout and stderr on PC from tests, I assume it should be the same on device? I would check before merging this, could be another bug